daily-test/pipeline.yaml

```diff
@@ -86,6 +86,11 @@ jobs:
       params:
         message: Load Generator Infrastructure Deploy Failed
         alert_type: failed
+    on_error:
+      put: slack-alert
+      params:
+        message: Load Generator Infrastructure Deploy Errored
+        alert_type: failed
```

Comment:
Not a blocker but would be nice to add `color: '#F4A624'` to make it match the colour in the UI 👀 

![image](https://user-images.githubusercontent.com/35296336/77649795-51ea2900-6f62-11ea-94a5-f00463da8af4.png)

Reply:
Good idea...